events_structs.go

```diff
@@ -156,6 +158,47 @@ func (ev *EventNewUserType) String() string {
 	return "You joined Scalingo. Hooray!"
 }
 
+type EventCreateReviewAppTypeData struct {
+	AppID             string `json:"app_id"`
+	ReviewAppName     string `json:"review_app_name"`
+	ReviewAppURL      string `json:"review_app_url"`
+	SourceRepoName    string `json:"source_repo_name"`
+	SourceRepoURL     string `json:"source_repo_url"`
+	PullRequestName   string `json:"pr_name"`
+	PullRequestNumber int    `json:"pr_number"`
+	PullRequestURL    string `json:"pr_url"`
+	ComesFromFork     bool   `json:"pr_comes_from_a_fork"`
+}
+
+type EventCreateReviewAppType struct {
+	Event
+	TypeData EventCreateReviewAppTypeData `json:"type_data"`
+}
+
+func (ev *EventCreateReviewAppType) String() string {
+	return fmt.Sprintf("the review app %s has been created from the pull request %s#%d", ev.TypeData.ReviewAppName, ev.TypeData.PullRequestName, ev.TypeData.PullRequestNumber)
```

Comment:
question: I don't know how it is displayed but by reading the format here, I would say the following is more readable:

```suggestion
	return fmt.Sprintf("the review app %s has been created from the pull request %s #%d", ev.TypeData.ReviewAppName, ev.TypeData.PullRequestName, ev.TypeData.PullRequestNumber)
```

Reply:
this is indeed more readable, fixing